flink-runtime-web/web-dashboard/src/app/pages/overview/overview.module.ts

```diff
@@ -22,9 +22,10 @@ import { ShareModule } from 'share/share.module';
 import { OverviewRoutingModule } from './overview-routing.module';
 import { OverviewComponent } from './overview.component';
 import { OverviewStatisticComponent } from './statistic/overview-statistic.component';
+import { TaskManagerItemComponent } from 'share/customize/task-manager-item/task-manager-item.component';
 
 @NgModule({
   imports: [CommonModule, ShareModule, OverviewRoutingModule],
-  declarations: [OverviewComponent, OverviewStatisticComponent]
+  declarations: [OverviewComponent, OverviewStatisticComponent, TaskManagerItemComponent]
```

Comment:
```suggestion
  declarations: [OverviewComponent, OverviewStatisticComponent]
```
TaskManagerItemComponent should be added in declarations and exports in `share.module.ts`